internal/proxy/proxy_test.go

```diff
@@ -257,6 +261,39 @@ func TestClientInitialization(t *testing.T) {
 				filepath.Join(testDir, pg, ".s.PGSQL.5432"),
 			},
 		},
+		{
+			desc: "with a Unix socket path per instance",
```

Comment:
Shall we add a unit test that shows how UnixSocketPath takes precedence over UnixSocket?

Reply:
Done.